app/models/accounts.py

```diff
@@ -26,14 +26,16 @@ def register(email, password):
             user = User.create(email=email, password=password)
             return user
 
-    @staticmethod
-    def authenticat(email, password):
-        user = User.get(email=email)
-        check = security.check_password_hash(user.password, password)
+    def authenticat(self, email, password):
```

Comment:
the logic is good but this would be much cleaner and readable
```
if self.email == email:
    return security.check_password_hash(self.password, password)
return False
```
the ```check_password hash``` returns True or False
and if the email doesn't check out it will return False anyway
also you don't need to return the user instance